--- a/misc/dtkgui.pc.in	2026-09-22 17:18:41.000000000 +0800
+++ b/misc/dtkgui.pc.in	2026-09-30 15:37:11.566278169 +0800
@@ -1,8 +1,8 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@LIBRARY_INSTALL_DIR@
-includedir=${prefix}/@INCLUDE_INSTALL_DIR@
-tooldir=${prefix}/@PACKAGE_TOOL_INSTALL_DIR@
+libdir=@LIBRARY_INSTALL_DIR@
+includedir=@INCLUDE_INSTALL_DIR@
+tooldir=@PACKAGE_TOOL_INSTALL_DIR@
 
 Name: dtk@DTK_NAME_SUFFIX@gui
 Description: Development ToolKit Gui Devel Library
